app/components/sidebars/main/channels_list/channels_list.js

```diff
@@ -150,16 +149,15 @@ export default class ChannelsList extends PureComponent {
                     onCancelButtonPress={this.onSearchCancel}
                     onChangeText={this.onSearch}
                     onFocus={this.onSearchFocused}
-                    searchIconCollapsedMargin={5}
-                    searchIconExpandedMargin={5}
                     keyboardAppearance={getKeyboardAppearanceFromTheme(theme)}
                     value={term}
+                    showCancel={false}
```

Comment:
shouldn't this be true? on Android is not showing the arrow button to cancel the search and there is no way to go back to the normal list without closing the sidebar

Reply:
You're right that this created a problem on Android, this was originally added to fix what @esethna mentioned on his review
> When the sidebar closes, we should "close" the jump to... field as well. ie if you hit jump to..., then close the sidebar and open it again, the jump to... search should not still be active.

I'll try to see if there's a way to accommodate what without breaking things on Android